src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -285,9 +289,9 @@ export class PaymentPageView extends View<CartDataType> {
         const orderBtn = document.querySelector<HTMLInputElement>(".summary-content__order-btn");
         orderBtn?.addEventListener('click', (event: Event) => {
 
-            if (validName && validAddress && validEmail &&
-                validTell && validCardName && validCardNumber &&
-                validCardDate && validCardCvv) {
+            if (valid.validName && valid.validAddress && valid.validEmail &&
+                valid.validTell && valid.validCardName && valid.validCardNumber &&
+                valid.validCardDate && valid.validCardCvv) {
```

Comment:
Please use func for it, for ex: 
const checkFormValidity = (valid) => {
...your code here
}